multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -391,18 +374,45 @@ write_files:
                     "args": [
                       "-machines=http://localhost:4001",
                       "-addr=0.0.0.0:53",
+                      "-ns-rotate=false", 
                       "-domain=cluster.local."
                     ],
-                    "image": "gcr.io/google_containers/skydns:2015-03-11-001",
+                    "image": "gcr.io/google_containers/skydns:2015-10-13-8c72f8c",
```

Comment:
Hard to tell here, but this container is being defined twice (and missing name field so apiserver complains). We should remove one